gluoncv/model_zoo/faster_rcnn/faster_rcnn.py

```diff
@@ -562,11 +562,11 @@ def faster_rcnn_resnet50_v1b_coco(pretrained=False, pretrained_base=True, **kwar
         features=features, top_features=top_features, classes=classes,
         short=800, max_size=1333, train_patterns=train_patterns,
         nms_thresh=0.5, nms_topk=-1, post_nms=-1,
-        roi_mode='align', roi_size=(14, 14), strides=16, clip=4.42,
+        roi_mode='align', roi_size=(14, 14), strides=16, clip=4.14,
```

Comment:
can we have the insight of this tweak?

Reply:
np.log(1000/16) as in detectron, but it does not seems to affect training that much